requirements/edx/development.txt

```diff
@@ -215,7 +215,7 @@ pbr==5.4.5                # via -r requirements/edx/testing.txt, stevedore
 pdfminer.six==20200402    # via -r requirements/edx/testing.txt
 piexif==1.1.3             # via -r requirements/edx/testing.txt
 pillow==7.1.1             # via -r requirements/edx/testing.txt, edx-enterprise, edx-organizations
-pip-tools==4.5.1          # via -r requirements/edx/pip-tools.txt
+pip-tools==5.0.0          # via -r requirements/edx/pip-tools.txt
```

Comment:
This makes me nervous without input from someone who better understands pip and pip-tools.  In my local venvs, I was always using pip 19.2.2, because pip 20 gave me problems, but now it is the minimum.  Support was added for pip>=20.1, so maybe that will resolve whatever issues I had?
Here is the changelog: https://github.com/jazzband/pip-tools/blob/master/CHANGELOG.md#500-2020-04-16

Reply:
We still have the configuration repo set to install pip 19.3.1 in all new containers and VMs, so we should probably update that before upgrading to the new pip-tools version.

Reply:
I guess this will need to be a coordinated change, because we might need to update pip to a version that is **only** supported by pip-tools 5.0.0?  I imagine we should constrain for now and add a ticket for this work?

Reply:
Yeah, constrain for now.  Supposedly pip 20.0.x should work with both pip-tools versions, but we'll see.  We were due for a pip upgrade anyway, would you mind writing the ticket for that?  Can be just a placeholder for now, I'll add some notes after the hackathon.

Reply:
Will do.